fuel-core/src/executor.rs

```diff
@@ -234,8 +240,8 @@ impl Executor {
             // TODO: reference the bytes directly from VM memory to save serialization. This isn't
             //       possible atm because the change output values are set on the tx instance in the vm
             //       and not also on the in-memory representation of the tx.
-            let tx_bytes = vm_result.tx().clone().to_bytes();
-            txs_merkle.push(&tx_bytes);
+            // let tx_bytes = vm_result.tx().clone().to_bytes();
+            // txs_merkle.push(&tx_bytes);
```

Comment:
This was actually different to the specs as it's meant to be a merkle tree of transaction ids and not just the bytes. Also now this is done in the `generate()` function.

Reply:
Whoops meant to post my other comment here instead

https://github.com/FuelLabs/fuel-core/pull/692/files#r993910492